### ISSUE TITLE
Running into issues at gf init

### ISSUE THREAD

**Issue by karllhughes** [goldfiglabs/introspector#1]
Hey Goldfig team, I'm trying out this repo for a tutorial I'm writing about maintaining AWS tags.

So far, I've got my containers running, but I run into the following error when I run `init`:

```
$ ./gf init
Checking for running goldfig
[2020-08-30 14:48:57,050] {bootstrap_db.py:85} ERROR - failed adding function from aws_logs_metricfilter_match.sql
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1246, in _execute_context
    cursor, statement, parameters, context
  File "/usr/local/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 588, in do_execute
    cursor.execute(statement, parameters)
psycopg2.errors.UndefinedObject: language "plv8" does not exist
...
```

There's [more to the trace here](https://pastebin.com/6mvPFjQT), but it's sounding like `plv8` isn't installed. 

I believe it's missing from the Postgres docker image, so I tried using this one in the `docker-compose.yml` file and it worked: https://registry.hub.docker.com/u/clkao/postgres-plv8/

*Update: Got it working with the other docker image.*

**Comment by gsoltis** [comment]
Hey @karllhughes I apparently didn't have notifications set up for issues! Sorry about that, and glad you got it working with Docker. We've actually since updated the docker-compose to point to one maintained by supabase that includes plv8.

**Comment by vikrum** [comment]
Fixed in 76e0123940bdc995f10afe6eb311a398bef4f5d6
